src/test/java/scm/address/logic/parser/AddressBookParserTest.java

```diff
@@ -193,6 +194,12 @@ public void parseCommand_addScheduleCommand() throws Exception {
         assertEquals(command, result);
     }
 
+    @Test
+    public void parseCommand_clearScheduleCommand() throws Exception {
+        assertTrue(parser.parseCommand(ClearScheduleCommand.COMMAND_WORD) instanceof ClearScheduleCommand);
+        assertTrue(parser.parseCommand(ClearScheduleCommand.COMMAND_WORD + " 10") instanceof ClearScheduleCommand);
```

Comment:
May I know why this is allowing + " 10" to parse? Should it mot throw an error?

Reply:
It doesn't throw an error, extra input after the command word is ignored in this case.

Reply:
@sarjinius is this an intended feature or a bug?

Reply:
Okay! Since `help 10` command also parses, I suppose you're not breaking the projects behaviour by allowing `clear_schedule 10` to pass!